port/ztpf/safe_storage.h

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Licensed Materials - Property of IBM
+ * "Restricted Materials of IBM"
+ *
+ * (c) Copyright IBM Corp. 2003, 2017 All Rights Reserved
+ *
+ * US Government Users Restricted Rights - Use, duplication or disclosure
+ * restricted by GSA ADP Schedule Contract with IBM Corp.
+ *******************************************************************************/

```

Comment:
You will need to update the license and amend your commit and force push it to the repo.